Create Malluma_manual.md

XuBoLi opened the pull request:
Introducation of the malluma  about installation , user and uninstallation.

XuBoLi committed:
Create Malluma_manual.md 

Introducation of the malluma  about installation , user and uninstallation.

xin3liang commented on doc/Malluma_manual.md:
```diff
@@ -0,0 +1,49 @@
+- **Introduction**
+- **Install guide**
+- **User guide**
+- **Uninstall guide**
```
目录的链接到章节加上去，参考https://raw.githubusercontent.com/XuBoLi/estuary/455fa2806a6a58d0a00506741bcc062bfeacc74b/doc/Packages_Manual.md

xin3liang commented on doc/Malluma_manual.md:
```diff
@@ -0,0 +1,49 @@
+## Introducation ##
+- The malluma software is the performance tuning tool, that could sample the cpu's data include the hardware events and software events by PMU, and scheduling condition's data, and I/O, network and so on. 
+- But it could only sample the ARM environment's data. It could help you  find your system bottleneck when you run your program. Maybe your program's bottleneck  is memory , I/O or scheduling, it could help you find  quickly.
+## Install guide ##
```
Install这章加上 estuary centos源怎么添加，如：
Please refer to https://github.com/open-estuary/distro-repo/blob/master/README.md CentOS repo adding part。

xin3liang commented on doc/Malluma_manual.md:
```diff
@@ -0,0 +1,49 @@
+For example, you has installded the malluma in the machine, it's ip is 192.168.1.100.
+You could visit the 192.16.1.100 by the browser (IE or Google chrome) to use the malluma's functions. When you visit it , you will be asked for the user/password.  
+That's default username is "admin", and the default password "Admin12#$". [Details info is here.](https://github.com/open-estuary/estuary/blob/master/doc/Malluma_UserGuide.pdf)
+## Uninstall guide##
```
"Uninstall guide##" 的##前应该有个空格

xin3liang commented on doc/Malluma_manual.md:
````diff
@@ -0,0 +1,49 @@
+```
+yum install Malluma-sec.aarch64
+```
+Second you should install the malluma software,  and specific steps are as follows:
````
这部分能否加在rpm脚本里面, 这个版本来不及的话，下个版本搞吧

xin3liang commented on doc/Malluma_manual.md:
```diff
@@ -0,0 +1,49 @@
+That's default username is "admin", and the default password "Admin12#$". [Details info is here.](https://github.com/open-estuary/estuary/blob/master/doc/Malluma_UserGuide.pdf)
+## Uninstall guide##
+
+First you should enter in the malluma's installed directory, then executing the commands as follows.
```
这步也是应该做到rpm脚本里面。这个版本改不了，下个版本搞吧。

XuBoLi committed:
Update the Malluma_manual.md

First insert the categorylinks.Second change the format aboput the catalog. Third, insert the repository link in the chapter of "Install guide".
